srcpkgs/djvulibre/template

```diff
@@ -1,16 +1,21 @@
 # Template file for 'djvulibre'
 pkgname=djvulibre
-version=3.5.27
-revision=4
+version=3.5.28
+revision=1
 build_style=gnu-configure
-hostmakedepends="pkg-config"
+configure_args="--host=${XBPS_TRIPLET}"
+hostmakedepends="pkg-config automake libtool"
 makedepends="tiff-devel"
 short_desc="Utilities for the DjVu image format"
 maintainer="Orphaned <orphan@voidlinux.org>"
-license="GPL-2"
+license="GPL-2.0-or-later"
 homepage="http://djvu.sourceforge.net/"
 distfiles="${SOURCEFORGE_SITE}/djvu/$pkgname-$version.tar.gz"
-checksum=e69668252565603875fb88500cde02bf93d12d48a3884e472696c896e81f505f
+checksum=82e392a9cccfee94fa604126c67f06dbc43ed5f9f0905d15b6c8164f83ed5655
+
+pre_configure() {
+	NOCONFIGURE=1 ./autogen.sh
```

Comment:
Most of our templates use `autoreconf -fi` directly. Is there a reason this shouldn't do that?

Did they stop distributing tarballs with `configure`? :/

Reply:
Yes, they stopped shipping `configure`.
There's a lot of templates calling `autogen.sh`, too. I think this is more robust because `autogen.sh` might do something more than just calling `autoreconf -fi` (not this case). But the two approaches are essentially the same.
I can change it if needed.

Reply:
There are others doing the same thing too, so it should be fine. Thanks